tests/e2e/hcp_external_auth_test.go

```diff
@@ -11,87 +11,187 @@ import (
-			Expect(err).ToNot(HaveOccurred())
-			Expect(output.ID).To(Equal(breakGlassCredID))
-			Expect(expirationTime.Format("Jan _2 2006 15:04 MST")).To(Equal(expiredTime))
-			Expect(output.Username).To(Equal(userName))
-			Expect(output.Status).To(Equal("issued"))
+		It("create/list/describe/delete HCP cluster with break_glass_credentials via Rosa client can work well - [id:72899]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Retrieve help for create/list/describe/delete break-glass-credential")
+				_, err := rosaClient.BreakGlassCredential.RetrieveHelpForCreate()
+				Expect(err).ToNot(HaveOccurred())
+				_, err = rosaClient.BreakGlassCredential.RetrieveHelpForDescribe()
+				Expect(err).ToNot(HaveOccurred())
+				_, err = rosaClient.BreakGlassCredential.RetrieveHelpForList()
+				Expect(err).ToNot(HaveOccurred())
+				_, err = rosaClient.BreakGlassCredential.RetrieveHelpForDelete()
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create a break-glass-credential to the cluster")
+				userName := common.GenerateRandomName("bgc-username", 2)
+				createTime := time.Now().UTC()
+				expiredTime := createTime.Add(2 * time.Hour).Format("Jan _2 2006 15:04 MST")
+
+				resp, err := rosaClient.BreakGlassCredential.CreateBreakGlassCredential(clusterID, userName, "--expiration", "2h")
+				Expect(err).ToNot(HaveOccurred())
+				textData := rosaClient.Parser.TextData.Input(resp).Parse().Tip()
+				Expect(textData).To(ContainSubstring("INFO: Successfully created a break glass credential for cluster '%s'", clusterID))
+
+				By("List the break-glass-credentials of the cluster")
+				breakGlassCredList, err := rosaClient.BreakGlassCredential.ListBreakGlassCredentialsAndReflect(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				isPresent, breakGlassCredential := breakGlassCredList.IsPresent(userName)
+				Expect(isPresent).To(BeTrue())
+				Eventually(rosaClient.BreakGlassCredential.WaitForBreakGlassCredentialToStatus(clusterID, "issued", userName), time.Minute*2, time.Second*10).Should(BeTrue())
+
+				By("Retrieve the break-glass-credential id")
+				breakGlassCredID := breakGlassCredential.ID
+
+				By("Describe the break-glass-credential")
+				output, err := rosaClient.BreakGlassCredential.DescribeBreakGlassCredentialsAndReflect(clusterID, breakGlassCredID)
+				Expect(err).ToNot(HaveOccurred())
+				// expiration timestamp changes from 4 to 5 seconds ahead, so have to remove second from time stamp
+				expirationTime, err := time.Parse("Jan _2 2006 15:04:05 MST", output.ExpireAt)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.ID).To(Equal(breakGlassCredID))
+				Expect(expirationTime.Format("Jan _2 2006 15:04 MST")).To(Equal(expiredTime))
+				Expect(output.Username).To(Equal(userName))
+				Expect(output.Status).To(Equal("issued"))
+
+				By("Get the issued credential")
+				_, err = rosaClient.BreakGlassCredential.GetIssuedCredential(clusterID, breakGlassCredID)
```

Comment:
Why not check the output for 'rosa list break-glass-credential'?

Reply:
I have checked in line number 64